lib/error.g

```diff
@@ -189,21 +189,50 @@ BIND_GLOBAL("ErrorInner",
     ERROR_COUNT := ERROR_COUNT+1;
     errorLVars := ErrorLVars;
     ErrorLVars := context;
-    # BreakOnError is defined by the `-T` command line flag in init.g
-    if QUITTING or not BreakOnError then
-        if not SilentErrors then
-            PrintTo("*errout*", "Error, ");
-            for x in earlyMessage do
-                PrintTo("*errout*", x);
-            od;
-            PrintTo("*errout*", "\n");
+    # The variables BreakOnError and QUITTING determine whether we
+    # start an interactive break loop or whether we use JUMP_TO_CATCH
+    # to exit this function.
+    # BreakOnError is defined by the `-T` command line flag in init.g.
+    doBreakLoop := BreakOnError and not QUITTING;
```

Comment:
`BreakOnError` is initialised depending on whether (and how many times?) `-T` is given on the command line, the user can (unfortunately) set it as well.

Reply:
I think it is quite fortunate that the user can do this, it has been crucial for some code I wrote, esp. for testing code.

Reply:
Good point. I'll change it to `initialize`.

Reply:
The unfortunate bit is that the user can set this to, say, a function which makes GAP crash when an error is hit.